packages/@lwc/shared/src/__tests__/html-attributes.spec.ts

```diff
@@ -4,45 +4,39 @@
  * SPDX-License-Identifier: MIT
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
-import { htmlAttributeToProperty, htmlPropertyToAttribute } from '../html-attributes';
+import { htmlPropertyToAttribute, kebabCaseToCamelCase } from '../html-attributes';
```

Comment:
I removed `htmlAttributeToProperty` in favor of `kebabCaseToCamelCase` so many of these tests could be removed